contributors/design-proposals/encryption.md

```diff
@@ -0,0 +1,205 @@
+# Encryption
+
+## Abstract
+
+The scope of this proposal is to ensure that resources can be encrypted at the datastore layer. Encrypting data over the network and via third party vendors is a desired feature but outside the scope of this proposal. There is future work to be done to enable end to end encryption, until then clients of the kubernetes API should be using TLS connections. Encryption will be optional for any resource but we suspect it will be used for the Secret resource in most cases.
```

Comment:
I think we should really strive to have the secret resource come encrypted by default instead of being optional. IMHO "we suspect it will be used for the Secret resource in most cases" isn't good enough.